README.rst

```diff
@@ -46,3 +46,7 @@ Create a standalone client::
 they would typically be on different machines and would not interfere with one another. Be aware
 of this when generating peer configs on a server node, or on any node that has a pre-existing
 wireguard config at the default file location.
+
+Version
+-------
+This is version 0.2.4
```

Comment:
This is not necessary to include in the readme. We can add a changelog file with this sort of information in it.

Reply:
Good idea